content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -96,10 +96,10 @@ Services will always have the `ready` condition set to `true`.
 
 #### Serving
 
-{{< feature-state for_k8s_version="v1.22" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="stable" >}}
 
-`serving` is identical to the `ready` condition, except it does not account for terminating states.
-Consumers of the EndpointSlice API should check this condition if they care about pod readiness while
+The `serving` condition is almost identical to the `ready` condition. The difference is that
```

Comment:
Hi @YashPimple .
The 'except' clause seems to help clarify the difference between the two conditions.

Is this correct:
The `serving` condition is nearly identical to the `ready` condition, except it does not account for terminating states. If a consumer of the EndpointSlice API cares about pod readiness while a pod terminates, check the `serving` condition.